doc/latex.rst

```diff
@@ -65,433 +49,487 @@ avoid interpretation as escape sequences, or use raw strings (as is done here).
+   ``'pdflatex'`` engine) to do:
+
+   .. code-block:: latex
+
+      \substitutefont{LGR}{\rmdefault}{cmr}
+      \substitutefont{LGR}{\sfdefault}{cmss}
+      \substitutefont{LGR}{\ttdefault}{cmtt}
+      \substitutefont{X2}{\rmdefault}{cmr}
+      \substitutefont{X2}{\sfdefault}{cmss}
+      \substitutefont{X2}{\ttdefault}{cmtt}
+
+   This is activated only under the condition that the ``'fontenc'`` key is
+   configured to load the ``LGR`` (Greek) and/or ``X2`` (Cyrillic)
+   pdflatex-font encodings (if the :confval:`language` is set to a Cyrillic
+   language, this ``'fontpkg'`` key must be used as "times" package has no
+   direct support for it; then keep only ``LGR`` lines from the above, if
+   support is needed for Greek in the text).
+
+   The ``\substitutefont`` command is from the eponymous LaTeX package, which
+   is loaded by Sphinx if needed (on Ubuntu Xenial it is part of
+   ``texlive-latex-extra`` which is a Sphinx requirement).
+
+   Only if the document actually does contain Unicode Greek letters (in text)
+   or Cyrillic letters, will the above default set-up cause additional
+   requirements for the PDF build. On Ubuntu Xenial, these are the
+   ``texlive-lang-greek``, ``texlive-lang-cyrillic``, and (with the above
+   choice of fonts) the ``cm-super`` (or ``cm-super-minimal``) packages.
+
+   For ``'xelatex'`` and ``'lualatex'``, the default is to use the FreeFont
+   family: this OpenType font family supports both Cyrillic and Greek scripts
+   and is available as separate Ubuntu Xenial package ``fonts-freefont-otf``.
+   It is not necessary to install the much larger ``texlive-fonts-extra``
+   package.
+
+   ``'platex'`` (Japanese documents) engine supports individual Cyrillic and
+   Greek letters with no need of extra user set-up.
+
+   :Default: ``'\\usepackage{times}'`` (or ``''`` when using a Cyrillic script)
+
+   .. versionchanged:: 1.2
+      Defaults to ``''`` when the :confval:`language` uses the Cyrillic
+      script.
+
+   .. versionchanged:: 2.0
+      Added support for individual Greek and Cyrillic letters:
+
+``'fncychap'``
+   Inclusion of the "fncychap" package (which makes fancy chapter titles),
+   default ``'\\usepackage[Bjarne]{fncychap}'`` for English documentation
+   (this option is slightly customized by Sphinx),
+   ``'\\usepackage[Sonny]{fncychap}'`` for internationalized docs (because
+   the "Bjarne" style uses numbers spelled out in English).  Other
+   "fncychap" styles you can try are "Lenny", "Glenn", "Conny", "Rejne" and
+   "Bjornstrup".  You can also set this to ``''`` to disable fncychap.
+
+   :Default: ``'\\usepackage[Bjarne]{fncychap}'`` for English documents,
+       ``'\\usepackage[Sonny]{fncychap}'`` for internationalized documents, and
+       ``''`` for Japanese documents.
+
+``'preamble'``
+   Additional preamble content.  One may move all needed macros into some file
+   :file:`mystyle.tex.txt` of the project source repertory, and get LaTeX to
+   import it at run time::
```

Comment:
I think second sentence is a tip to make preamble simple. So it would be better to separate from main description of preamble field. cc: @jfbu